src/templates/aws/addons/ecs.ts

```diff
@@ -75,13 +81,19 @@ const ecsModuleContent = dedent`
     ecr_tag                            = var.ecr_tag
     security_groups                    = module.security_group.ecs_security_group_ids
     alb_target_group_arn               = module.alb.alb_target_group_arn
-    aws_cloudwatch_log_group_name      = module.log.aws_cloudwatch_log_group_name
-    desired_count                      = var.ecs.task_desired_count
-    cpu                                = var.ecs.task_cpu
-    memory                             = var.ecs.task_memory
-    deployment_maximum_percent         = var.ecs.deployment_maximum_percent
-    deployment_minimum_healthy_percent = var.ecs.deployment_minimum_healthy_percent
-    container_memory                   = var.ecs.task_container_memory
+    aws_cloudwatch_log_group_name      = module.cloudwatch.aws_cloudwatch_log_group_name
```

Comment:
This fixes the incorrect module name reference as it was defined as "cloudwatch" instead of "log"

https://github.com/nimblehq/infrastructure-templates/blob/c7b3877f30f8dd9284f7f4bafafebe165ca1e67b/src/templates/aws/addons/cloudwatch.ts#L7-L8